reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -909,6 +919,17 @@ def run_prod(job):
                     job.sp.pressure,
                 )
             )
+            with open("production_information.txt", "w") as text_file:
+                prod_file = text_file.write(
+                    "All prod replicates done. Simulation finished for {} molecule = {}, ensemble = {}, temperature= {} K, pressure = {} kPa.".format(
+                        job,
+                        job.sp.molecule,
+                        job.sp.ensemble,
+                        job.sp.temperature,
+                        job.sp.pressure,
+                    )
+                )
+            text_file.close()
```

Comment:
The `with` statement will ensure that the file is closed automagically, no need to close a file manually when using them!
```suggestion
```